application/models/esmodel.py

```diff
@@ -7,7 +7,7 @@
 from application.errors import NotFound, NotInitialized
 
 
-class ESModel():
+class ESModel:
```

Comment:
Why? Good chance for me to learn more python syntax :D

Reply:
Braces are for inheritance, if you don't inherit there is no need for braces. At least that's what PyCharm suggested.